src/toNestError.ts

```diff
@@ -13,14 +13,29 @@ export const toNestError = <TFieldValues>(
 ): FieldErrors<TFieldValues> => {
   options.shouldUseNativeValidation && validateFieldsNatively(errors, options);
 
+  const entries = Object.entries(errors)
+      .sort(([leftPath], [rightPath]) => {
+        if (leftPath === rightPath) {
+          return 0;
+        }
+
+        return leftPath > rightPath ? -1 : 1
+      });
```

Comment:
This logic worries me because this function is a generic function and this will impact other resolvers as well.